src/LibraryManager.Vsix/Commands/CommandTable/LocalizationFiles/VSCommandTable.pt-BR.xlf

```diff
@@ -22,51 +22,33 @@
     <body>
       <group id="E:\LibraryManager\src\LibraryManager.Vsix\Commands\CommandTable\VSCommandTable.en.vsct">
         <trans-unit id="InstallPackage|ButtonText">
-          <source xml:lang="en">Add Client-Side Library...</source>
-          <target xml:lang="pt-BR" state="needs-review-translation" state-qualifier="mt-suggestion">Adicione a biblioteca de cliente...</target>
-          <alt-trans match-quality="100%" tool="BlackBox/MSR MT">
-            <target xml:lang="pt-BR" state-qualifier="mt-suggestion">Adicione a biblioteca de cliente...</target>
-          </alt-trans>
+          <source xml:lang="en">Add Client-Side Library...</source><target state="translated">Adicionar biblioteca do lado do cliente...</target>
```

Comment:
Do we have to translate client side? I think it makes more sense "Adicionar bibliotecas client-side"